deps/3rdparty/libdispatch/build/libdispatch.vcxproj

```diff
@@ -249,19 +246,13 @@
     <ClangCompile Include="..\src\shims\mach.c" />
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
   <PropertyGroup>
-    <PrebuiltPath Condition="'$(PrebuiltPath)' == ''">$(WINOBJC_SDK_ROOT)\deps\prebuilt\Universal Windows\$(PlatformTarget)\$(Configuration)\\</PrebuiltPath>
+    <PrebuiltRoot Condition="'$(PrebuiltRoot)' == ''">$(WINOBJC_SDK_ROOT)\tools\deps\prebuilt\\</PrebuiltRoot>
+    <PrebuiltPath Condition="'$(PrebuiltPath)' == ''">$(PrebuiltRoot)\Universal Windows\$(PlatformTarget)\$(Configuration)\\</PrebuiltPath>
```

Comment:
>PrebuiltPath [](start = 5, length = 12)

I don't see this used. Whats the deal?

Reply:
Yeah, the target that used them was removed. This target copies over checked in files in the main repo. In an ideological sense, this is bad. We don't want to have any 3rdparty projects copying over files checked into the main repo.